android/src/com/mapswithme/maps/editor/ViralFragment.java

```diff
@@ -66,6 +68,8 @@ public void onClick(View v)
         dismiss();
       }
     });
+    Statistics.INSTANCE.trackEvent(Statistics.EventName.EDITOR_SHARE_SHOW,
+                                   Statistics.params().add("showed", sViralText.equals(viralChangesMsg) ? "change" : "rating"));
```

Comment:
Можно бы и забить, но я просто уберу статик у текста, всё равно он показывается ровно один раз.
> On Apr 21, 2016, at 3:57 PM, Alexander Marchuk <notifications@github.com> wrote:
> 
> In android/src/com/mapswithme/maps/editor/ViralFragment.java <https://github.com/mapsme/omim/pull/2948#discussion_r60574326>:
> 
> > @@ -66,6 +68,8 @@ public void onClick(View v)
> >          dismiss();
> >        }
> >      });
> > +    Statistics.INSTANCE.trackEvent(Statistics.EventName.EDITOR_SHARE_SHOW,
> > +                                   Statistics.params().add("showed", sViralText.equals(viralChangesMsg) ? "change" : "rating"));
> Условие equals так же ложно, если перед показом диалога, но после инициализации sViralText поменялась локаль в системе. Или забиваем на это?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/mapsme/omim/pull/2948/files/631432490217013f235baa987532d4e073dcca69#r60574326>